sphinx/writers/latex.py

```diff
@@ -737,13 +736,15 @@ def visit_desc_signature(self, node: Element) -> None:
         if not node.get('is_multiline'):
             self._visit_signature_line(node)
         else:
-            self.body.append('%\n\\pysigstartmultiline\n')
+            self.body.append('%\n')
+            self.body.append('\\pysigstartmultiline\n')
```

Comment:
I'll replace '\n` by constant on the next step. It allows us to use raw string for LaTeX macros.